core/src/components/input/input.ionic.outline.scss

```diff
@@ -119,3 +119,19 @@
 :host(.input-fill-outline) ::slotted([slot="end"]) {
   margin-inline-start: 8px;
 }
+
+// Input - Disabled
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline.input-disabled) {
+  // color for the text within the input
+  --color: #{$ionic-color-neutral-400};
+  --background: transparent;
+}
+
+:host(.input-fill-outline.input-disabled) .native-wrapper {
+  --background: #{$ionic-color-neutral-10};
+  border-radius: var(--border-radius);
```

Comment:
Should this rule just be applied to `.native-wrapper` or `.input-wrapper` like [`input.scss`](https://github.com/ionic-team/ionic-framework/blob/62a3e115ef87cc418df7a93e059a5d1964c24620/core/src/components/input/input.scss#L233)?

Reply:
No, because the `input-wrapper` also includes the label. If we add it, then the styles will also show within the label area like this:

![Screenshot 2024-04-15 at 11 50 46 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/599a93fe-056c-48f4-96a7-5ecbc5609a04)

Outline only has the background and border within the native input.

Reply:
Oh I see. Can we use `border-radius: inherit` instead?

Reply:
Worked: https://github.com/ionic-team/ionic-framework/pull/29323/commits/d1f00990ae38b415e416f75fe9a635a15003104f